docs/GeneralSetup.md

```diff
@@ -42,6 +42,80 @@ ALTER TABLE ['{table_name}'] ALTER COLUMN ['{primary_key_column_name}'] int NOT
 ALTER TABLE ['{table_name}'] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (['{primary_key_column_name}']);
 ```
 
+## Create Login and User
+
+SQL bindings connect to the target database by using a Connection String configured in the app settings. This will require a login be created that the function will use to access the server.
+
+For local testing and development using a SQL (username/password) or Azure Active Directory Login is typically the easiest, but for deployed function apps it is recommended to use [Azure Active Directory Managed Authentication](https://learn.microsoft.com/azure/azure-functions/functions-identity-access-azure-sql-with-managed-identity).
+
+## Assign Permissions
+
+The login used by the function will need to have the following permissions assigned to the user it's mapped to in order for it to function. The permissions required for each type of binding is listed below.
```

Comment:
nit: using function as a verb in docs talking about the function noun, suggesting revised wording for clarity

The login used by the function will need to have the following permissions assigned to the user it's mapped to in order for it to _function_.

-->

The login used by the function will need to have the following permissions assigned to the database user it's mapped to in order for it to successfully interact with the database.